lib/beaker/host/unix/pkg.rb

```diff
@@ -179,6 +179,8 @@ def upgrade_package(name, cmdline_args = '', opts = {})
     'ubuntu-10.04-i386'  => 'lucid',
     'ubuntu-12.04-amd64' => 'precise',
     'ubuntu-12.04-i386'  => 'precise',
+    'ubuntu-14.04-amd64' => 'trusty',
+    'ubuntu-14.04-i386'  => 'trusty',
```

Comment:
This same info, in a slightly different format, is in platform.rb - maybe we should fix that :)